src/components/ModelizerModelView.vue

```diff
@@ -141,14 +161,64 @@ async function updatePluginsAndTemplates() {
       templates.value = response;
     })
     .catch(() => {
-      Notify.create({
-        type: 'negative',
-        message: t('errors.templates.getData'),
-        html: true,
-      });
+      notifyError();
     });
 }
 
+/**
+ * Required form compatibility with the HTML drag and drop API.
+ *
+ * @param {DragEvent} event - The drag event.
+ */
+function dragOverHandler(event) {
+  event.preventDefault();
+}
+
+/**
+ * Instantiate from a dragged component definition or template.
+ *
+ * @param {DragEvent} event - The drag event.
+ */
+async function dropHandler(event) {
```

Comment:
unnecessary : https://jsdoc.app/tags-async.html

Reply:
Good point @sohzaz but you don't add it in the documentation :/
So add it and we will be ok ;)

Reply:
Sorry, I update my previous comment, but like the link you give, you still have to add the @return tag with explanation

Reply:
Even the tag itself is unnecessary in this situation.
From the linked doc: ``In general, you do not need to use this tag, because JSDoc automatically detects asynchronous functions and identifies them in the generated documentation.``

Reply:
better be like all other functions :
`* @return {Promise<void>} Promise with nothing on success otherwise an error.`

Reply:
updated